### ISSUE TITLE
bug: fig autocomplete stops working when triggering Alfred's clipboard expander

### ISSUE THREAD

**Issue by golaniy** [withfig/fig#2467]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

autocomplete window to show up after using alfred to fill in one of the parameters

### Actual behaviour

once pasting something from alfred, autocomplete window stops showing up

### Steps to reproduce

In main description

### Environment

```yaml
fig-details:
  - 2.15.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M2
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /Applications/Fig.app/Contents/MacOS/fig_desktop
  - terminal: iterm
  - cwd: /
  - exe-path: /Applications/Fig.app/Contents/MacOS/fig-darwin-universal
  - install-method: brew
  - env-vars:
    - FIG_TERM: 2.15.0
    - __CFBundleIdentifier: com.mschrage.fig
    - FIG_SET_PARENT_CHECK: 1
    - PATH: /Users/yotamg/opt/anaconda3/envs/pywiliot_develop/bin:/Users/yotamg/opt/anaconda3/condabin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/yotamg/.fig/bin:/Users/yotamg/.local/bin
    - TERM: xterm-256color
    - FIG_SET_PARENT: b1907efa-6b19-482d-a812-829342ee0c62
    - FIG_PID: 35700
    - SHELL: /bin/zsh
    - FIGTERM_SESSION_ID: b1907efa-6b19-482d-a812-829342ee0c62
```

**Comment by mschrage** [comment]
Hey @golaniy, can you record a video reproducing this behavior?

**Comment by golaniy** [comment]
https://user-images.githubusercontent.com/6959704/235651832-82533779-4fa1-4502-a0e6-1a3b5e3af484.mov

**Comment by mschrage** [comment]
Thanks for posting this video! We should have a fix in the next release. (This also helped us fix #2271)
